admin/howto/configurator.md

```diff
@@ -0,0 +1,74 @@
+# Configure the hub as an admin with the Configurator
+
+The [JupyterHub configurator](https://github.com/yuvipanda/jupyterhub-configurator)
+allows admins to change a subset of hub settings without requiring involvement
+from 2i2c engineers.
+
+```{warning}
+Be careful while changing these settings! We don't expose anything that can
+make the hub inaccessible to *admins*, but you can still change things that
+break user server starts. You can always fix it by undoing the change that broke
+it, and a way to see the 'version history' of your config changes is in the
+works.
+```
+
+## Accessing the Configurator
```

Comment:
Just a note that we have a shift in tense for our headers. The title is "Configure", while the headers are "`-ing`" words (Access_ing_, Mak_ing_, etc). Could we keep them all the same active tense? `Access the Configurator`, `Make changes with the configurator`?

Reply:
I picked 'Access', since it matches the title's 'configure', which completes a 'how do I...?'